puppet/services/opendaylight-api.yaml

```diff
@@ -32,6 +36,14 @@ parameters:
   OpenDaylightApiVirtualIP:
     type: string
     default: ''
+  OpendaylightPortBindingController:
+    description: Opendaylight port binding controller
+    type: string
+    default: 'pseudo-agentdb-binding'
```

Comment:
Note I think this has a bigger impact when you look at OVS. That then requires we use hostconfig for OVS...we need to determine the impacts of this change for OVS and upstream.  Perhaps this default should be network-topology until we are ready to move to hostconfig for OVS.